examples/sendToAndTransferFromStealthAddress/components/stealth-actions-example.tsx

```diff
@@ -0,0 +1,270 @@
+import {
+  type GenerateStealthAddressReturnType,
+  VALID_SCHEME_ID,
+  computeStealthKey,
+  generateKeysFromSignature,
+  generateStealthAddress,
+  generateStealthMetaAddressFromSignature
+} from '@scopelift/stealth-address-sdk';
+import React, { useEffect, useState } from 'react';
+import {
+  http,
+  type Address,
+  createWalletClient,
+  custom,
+  formatEther,
+  parseEther
+} from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { sepolia } from 'viem/chains';
+import 'viem/window';
+import { useBalance, useWaitForTransactionReceipt } from 'wagmi';
+import { RPC_URL } from '..';
+
+const StealthActionsExample = () => {
+  if (!window.ethereum) throw new Error('window.ethereum is required');
+
+  const CHAIN = sepolia;
+  const SCHEME_ID = VALID_SCHEME_ID.SCHEME_ID_1;
+  const MESSAGE_TO_SIGN = `Generate Stealth Meta-Address on ${CHAIN.id} chain`;
+  const SEND_AMOUNT = '0.01';
+  const WITHDRAW_AMOUNT = '0.005';
```

Comment:
nit: could we make this `0.01` or let users withdraw twice?

Reply:
a lot of sepolia eth poor devs out there

Reply:
yee true:

https://github.com/ScopeLift/stealth-address-sdk/pull/77/commits/e2ebb344b3a5e42a027e7c7bc63e32555569f0ac